app/forms/idv_profile_form.rb

```diff
@@ -0,0 +1,48 @@
+class IdvProfileForm
+  attr_reader :user, :errors, :params
+
+  def initialize(user)
+    @user = user
+    @errors = []
+  end
+
+  def submit(params)
+    @params = params
+    error_required_params if required_params_blank?
+    error_duplicate_ssn if ssn_taken?
+    self
```

Comment:
Is returning self a normal thing to do in a submit method?  Other places I see `submit` return true and success and false on failure.